snark-verifier/Cargo.toml

```diff
@@ -30,7 +30,7 @@ bytes = { version = "1.2", optional = true }
 rlp = { version = "0.5", default-features = false, features = ["std"], optional = true }
 
 # loader_halo2
-halo2-ecc = { git = "https://github.com/scroll-tech/halo2-lib", branch = "develop", optional = true, default-features=false, features=["halo2-pse","display"] }
+halo2-ecc = { git = "https://github.com/scroll-tech/halo2-lib", branch = "ecc_double_p256", optional = true, default-features=false, features=["halo2-pse","display"] }
```

Comment:
reminder to update once https://github.com/scroll-tech/halo2-lib/pull/31 is merged